flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/service/NativeFlinkServiceTest.java

```diff
@@ -180,8 +186,10 @@ public void testCancelJobWithLastStateUpgradeMode() throws Exception {
                         .get());
     }
 
-    @Test
-    public void testTriggerSavepoint() throws Exception {
+    @ParameterizedTest
+    @MethodSource("provideSavepointFormatType")
+    public void testTriggerSavepoint(Configuration configuration, FlinkConfigManager configManager)
+            throws Exception {
```

Comment:
These tests do not actually test that the correct savepoint format type is used during savepointing.

We should remove these parameterized tests and simply validate that if the user configures the savepoint type the rest api is called with that type in the FlinkService